docs/plans/2026-05-01-track-b-severity-change-reason.md

```
@@ -0,0 +1,733 @@
+    }"#;
+    let trace: CalibratorTraceEntry = serde_json::from_str(pre_track_b).unwrap();
+    assert!(trace.severity_change_reason.is_none());
+}
+```
+
+**Step 2: Run tests to verify they fail**
+
+```bash
+cd /Users/jsnyder/Sources/github.com/jsnyder/quorum/.worktrees/calibrator-trace-reason
+cargo test --bin quorum -p quorum severity_change_reason_serializes_snake_case trace_entry_omits_reason_when_none trace_entry_deserializes_pre_track_b_lines 2>&1 | tail -10
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)